src/partners/swapuz.ts

```diff
@@ -158,17 +158,16 @@ export const querySwapuz = async (
           )
           done = true
         }
-      } catch (e) {
-        const err: any = e
-        datelog(err.message)
       }
-    }
-    // console.log(
-    //   `Swapuz page=${page}/${maxPage} oldestIsoDate: ${oldestIsoDate}`
-    // )
+      datelog(`Swapuz page=${page}/${maxPage} oldestIsoDate: ${oldestIsoDate}`)
 
-    if (currentPage >= maxPage) {
-      break
+      if (currentPage >= maxPage) {
+        break
+      }
+    } catch (e) {
+      const err: any = e
```

Comment:
You can put the `any` type right inside of the `catch` block, instead of using the two-line hack:

```ts
try {
} catch (e: any) {
}
```

Reply:
For some reason. VSCode auto corrects `(e: any)` and changes it back to (e)